src/robusta/core/sinks/telegram/telegram_client.py

```diff
@@ -9,14 +9,16 @@
 
 
 class TelegramClient:
-    def __init__(self, chat_id: int, bot_token: str):
+    def __init__(self, chat_id: int, thread_id: int, bot_token: str):
         self.chat_id = chat_id
+        self.thread_id = thread_id
         self.bot_token = bot_token
 
     def send_message(self, message: str, disable_links_preview: bool = True):
         url = f"{TELEGRAM_BASE_URL}/bot{self.bot_token}/sendMessage"
         message_json = {
             "chat_id": self.chat_id,
+            "message_thread_id": self.thread_id,
```

Comment:
Just to be sure, if the `thread_id` is `None`, does this fallback to the default behavior? 
Did you test it with an empt `thread_id`?

Reply:
Yep, I've successfully tested this with an existing Robusta configuration that contains several Telegram sinks without the `thread_id` parameter set.

Reply:
Thank you!